http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -762,15 +752,16 @@ class ClusterMonitorActor(
 
   private def recordStatusTransitionMetrics(clusterUI: RuntimeUI,
                                             origStatus: RuntimeStatus,
-                                            finalStatus: RuntimeStatus): IO[Unit] =
+                                            finalStatus: RuntimeStatus,
+                                            cloudService: CloudService): IO[Unit] =
     for {
       endTime <- IO(System.currentTimeMillis)
-      baseName = s"ClusterMonitor/${clusterUI.asString}/${origStatus}->${finalStatus}"
-      counterName = s"${baseName}/count"
-      timerName = s"${baseName}/timer"
+      metricsName = s"monitor/${clusterUI.asString}/transition/${origStatus}_to_${finalStatus}"
       duration = (endTime - startTime).millis
-      _ <- metrics.incrementCounter(counterName).runAsync(_ => IO.unit).toIO
-      _ <- metrics.recordResponseTime(timerName, duration).runAsync(_ => IO.unit).toIO
+      tags = Map("cloudService" -> cloudService.asString, "ui_client" -> clusterUI.asString)
+      _ <- openTelemetryMetrics.incrementCounter(metricsName, 1, tags)
+      histoBuckets = List(30000.0, 40000.0, 50000.0, 60000.0, 72000.0, 90000.0, 120000.0, 150000.0, 180000.0, 210000.0, 240000.0, 270000) //histogram buckets from 1 min to 6 min
```

Comment:
May want to add to comments why 1 min to 6 mins. Also what the increment pattern is and why.

Could `histoBuckets` be more strongly typed? Could perhaps use a time-related library? It may also make reading easier with units.

May be worth spelling out `histogram` as `histo` may not be an obvious short form :)

`histoBuckets` is used in two places. Does it make sense to factor it out of the methods so it's defined in a single place?

Reply:
they're 2 different buckets. Buckets are metrics specific, like starting -> running should have different buckets from runtimeCreation

 I thought about making them `FiniteDuration`, but I think millis is a bit more granular.

Reply:
> they're 2 different buckets.

Sorry, not sure how I missed that.

Reply:
Looks like it's 30 seconds to 4.5 minutes.. agreed it would be good to break this variable out to another place (`package.scala`?) and add a comment.

Reply:
this is not really being reused...each metric should have their own distribution